Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -449,6 +449,7 @@ def types_that_cannot_be_forward_declared():
         'PlatformXR::SessionMode',
         'PlatformXR::VisibilityState',
         'String',
+        'SystemMemoryPressureStatus',
```

Comment:
Why do we have to do this? Why can't this enum be forward-declared? This usually works.

Reply:
We don't need this for `WTFLogLevel` for example. I suspect there is a better way to fix this.

Reply:
`WTFLogLevel` is defined at the global scope. Please read the error message. All I need is this single line https://github.com/WebKit/WebKit/blob/77f02397ad6c9e34e93941a9967b2d6c21e4c1a7/Source/WTF/wtf/MemoryPressureHandler.h#L282

Reply:
Please let me know if the build is still broken for you after https://commits.webkit.org/285454@main. It fixed the build for me (I was seeing the same error as you locally on Mac)

Reply:
Still same error message.